games-fps/gzdoom/gzdoom-4.1.2.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils desktop xdg
+
+DESCRIPTION="A modder-friendly OpenGL source port based on the DOOM engine"
+HOMEPAGE="https://zdoom.org"
+SRC_URI="https://github.com/coelckers/${PN}/archive/g${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD BZIP2 DUMB-0.9.2 GPL-3 LGPL-3 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="fluidsynth gtk openal openmp"
+
+DEPEND="
+	media-libs/libsdl2[opengl]
+	sys-libs/zlib
+	virtual/jpeg:0
+	gtk? ( <x11-libs/gtk+-4:= )"
+RDEPEND="
+	${DEPEND}
+	fluidsynth? ( media-sound/fluidsynth )
+	openal? (
+		media-libs/libsndfile
+		media-libs/openal
+		media-sound/mpg123
+	)"
+
+S="${WORKDIR}/${PN}-g${PV}"
+
+PATCHES="${FILESDIR}/${PV}"
+
+src_prepare() {
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DINSTALL_DOCS_PATH="${EPREFIX}/usr/share/doc/${P}/docs"
```

Comment:
I see that the [20151213 Gentoo council meeting summary](https://projects.gentoo.org/council/meeting-logs/20151213-summary.txt) states:

> The council voted in favor of deprecating the /usr/games and /etc/games
directories. Games packages should _not_ install any files there, but
follow the normal guidelines for install locations instead. Two
exceptions are made: (a) Games packages can install files in
/usr/share/games (instead of /usr/share) if that is the location used
by upstream. (b) Shared high-score or game state files can be placed
in /var/games or a subdirectory of it.

`/usr/share/games` is used by GZDoom upstream. Since this is one of the exceptions allowed for game packages, should we permit GZDoom to install files there to keep it in sync with upstream?

Reply:
Are there any drawbacks in placing these files to `/usr/share/doom`? I see that by default the program searches the files in both directories:
```
[FileSearch.Directories]
Path=$HOME/.config/gzdoom
Path=/usr/local/share/
Path=/usr/local/share/doom
Path=/usr/local/share/games/doom
Path=/usr/share/doom
Path=/usr/share/games/doom
Path=$DOOMWADDIR
```
Or have I missed something?

Reply:
Ah, you are correct. In that case `/usr/share/doom` shouldn't be much of a problem. The latest force push I did should be complete then. If there are any problems you notice, let me know.